### ISSUE TITLE
defnet not present in awesome-defold

### ISSUE THREAD

**Issue by Jerakin** [britzl/defnet#6]

**Comment by britzl** [comment]
Yeah, I know, but I'm not really happy with the repo and the content. Is it useful enough? Is it stable enough?

**Comment by Jerakin** [comment]
Maybe depends what it's used for? We just merged a feature into Pokedex5e that enables people to send their Pokemon over the network. It have worked fine for that (so far) but might be to early to say if there will be any hick-ups.

Here is the PR that adds it to the project. https://github.com/Jerakin/Pokedex5E/pull/493 Notably copied p2p_discovery into the project because he needed quite a few changes done to it.

I guess we will see in a few days or a week if the users complains about it not always working :)
